Errors translation 📰 (#11)
bampakoa/angular-gr: aio/content/errors/NG0201.md

@shortDescription No provider for {token} found!

@description
Αυτό το σφάλμα εμφανίζεται όταν προσπαθείτε να εισάγετε μια υπηρεσία αλλά δεν έχετε δηλώσει τον αντίστοιχο πάροχο. Ένας πάροχος είναι μια αντιστοίχιση που παρέχει μια τιμή που μπορείτε να εισάγετε στον κατασκευαστή μιας κλάσης στην εφαρμογή σας.

Διαβάστε περισσότερα για τους παρόχους [Dependency Injection guide](guide/dependency-injection).

@debugging

> Εργαστείτε προς τα πίσω από το αντικείμενο όπου το σφάλμα δηλώνει ότι ο [πάροχος](guide/architecture-services) λείπει: `No provider for ${this}!`. Αυτό συνήθως συμβαίνει σε [υπηρεσίες](tutorial/toh-pt4), που απαιτούν ανύπαρκτους παρόχους.

Reviewer: ```suggestion
Ανατρέξτε προς τα πίσω από το αντικείμενο όπου το σφάλμα δηλώνει ότι ο [provider](guide/architecture-services) λείπει: `No provider for ${this}!`. Αυτό συνήθως συμβαίνει σε [services](tutorial/toh-pt4), που απαιτούν providers που δεν υπάρχουν.
```

> Για να διορθώσετε το σφάλμα, βεβαιωθείτε ότι η υπηρεσία σας είναι εγγεγραμμένη στη λίστα παρόχων ενός `NgModule` ή ότι έχει το διακοσμητή `@Injectable` με την ιδιότητα `providedIn` στην κορυφή.

Reviewer: ```suggestion
Για να διορθώσετε το σφάλμα, βεβαιωθείτε ότι το service σας είναι καταχωρημένο στη λίστα των providers ενός `NgModule` ή ότι έχει τον decorator `@Injectable` με την ιδιότητα `providedIn` στην κορυφή.
```

> Η πιο συνηθισμένη λύση είναι να προσθέσετε έναν πάροχο στο `@Injectable` χρησιμοποιώντας το `providedIn`:

Reviewer: ```suggestion
Η πιο συνηθισμένη λύση είναι να προσθέσετε έναν provider στο `@Injectable` χρησιμοποιώντας το `providedIn`:
```
